deepmd/loss/ener.py

```diff
@@ -252,6 +252,340 @@ def eval(self, sess, feed_dict, natoms):
         return results
 
 
+class EnerSpinLoss(Loss):
```

Comment:
It looks that this class just add `use_spin` compared to `EnerLoss`. Why not directly add this option to `EnerLoss`?

Reply:
Force is divided into atomic force and magnetic force, and their losses are built individually. So, we need `EnerSpinLoss`.